### PR TITLE
feat(atenlib):add ops(scatter-add, floor)

### PR DESCRIPTION
**Pull request by xiaowuhu**
ORT failed on FullGraph test mode, because if-else branch have different shape (rank 0 vs. rank N)

### CODE REVIEW

File: onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py [hunk 1]
```diff
@@ -337,6 +337,13 @@ def _permute_input_wrangler(
     return args, kwargs
 
 
+def _scatter_add_input_wrangler(
+    args: list[Any], kwargs: dict[str, Any]
+) -> tuple[list[Any], dict[str, Any]]:
+    kwargs["dim"] = args.pop(1)
```
Comment (justinchuby) [inline onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py:343]: Curious why this is needed?

Comment (xiaowuhu) [reply onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py:343]: The original test case is:  self(tensor), dim( [int, int, ...]), index(tensor), src(tensor). 
so the dim in the second place, cannot not be parsed.
if we use wrangler to change it to Tensor, the follow op.ScatterElements(self, index, src, axis=dim) will be failed, due to dim is not an attribute.

File: onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py [hunk 2]
```diff
@@ -671,6 +680,11 @@ def _where_input_wrangler(
     xfail(
         "round", variant_name="decimals_neg_3", reason="The op does not support decimals yet"
     ),
+    xfail(
+        "scatter_add",
+        reason="fixme: ORT failed due to different rank for if-else branch",
```
[the remark this answers is not in the document]
Comment (xiaowuhu) [reply onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py]: skipped

File: onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py [hunk 3]
```diff
@@ -868,6 +863,11 @@ def _where_input_wrangler(
         matcher=lambda sample: len(sample.args[0]) == 0,
         reason="Empty perm is not supported",
     ),
+    skip(
+        "scatter_add",
+        matcher=lambda sample: len(sample.input.shape) == 0,
+        reason="Rank(0) input will lead ORT failed due to different rank(result) in if-else branch",
```
Comment (justinchuby) [inline onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py:880]: fixme:

**Review by justinchuby — APPROVED** [review]

**Pull request merged**